simulation_lab/BaseTestCaseView.py

```diff
@@ -12,8 +12,7 @@ def assertMessageLevel(self, response, messageLevel):
 
     def createLoggedInUser(self, isAdmin):
         if isAdmin:
-            User.objects.create_superuser("test@test.com", "Test1234")
+            User.objects.create_user(email="test@test.com", password="Test1234", is_superuser=True)
```

Comment:
I think this functionality is a little different from create_superuser(). I don't  think is_staff will be set to true. Not sure what other differences there are

Reply:
I suggest still using create_superuser() but no  strong feelings